src/07-Testing.mdx

```diff
@@ -0,0 +1,86 @@
+---
+name: 07. Testing
+---
+
+import Authors from './helpers/Authors';
+import sloth from './assets/sloth.jpg'
+
+# Testing: Basic Unit Testing
+Level: *Useful Sloth*
```

Comment:
Can you make the level name an alliteration?

Reply:
Subsidiary Sloth
Serviceable Sloth

Can't think of anything else...

Reply:
I’m fine with both go ahead and choose one 🙌🏻